internal/guest/runtime/runc/container.go

```diff
@@ -366,6 +370,26 @@ func (c *container) startProcess(
 	cmd := runcCommandLog(logPath, args...)
 
 	var pipeRelay *stdio.PipeRelay
+	var fifoPipe *os.File
+
+	pipeName, exists := annotations["io.microsoft.bmc.logging.pipelocation"]
```

Comment:
Please add a comment above this code block explaining that this is an experimental feature and has not been stabilized yet.

Reply:
added comment at the beginning of the section